backend/webapi/Features/Credentials/BCProvider.ResetMFA.cs

```diff
@@ -0,0 +1,69 @@
+namespace Pidp.Features.Credentials;
+
+using System.Text.Json.Serialization;
+using DomainResults.Common;
+using FluentValidation;
+using HybridModelBinding;
+using Microsoft.EntityFrameworkCore;
+using Pidp.Data;
+using Pidp.Infrastructure.Auth;
+using Pidp.Infrastructure.HttpClients.BCProvider;
+
+public class BCProviderResetMFA
```

Comment:
The C# naming conventions is to have only the leading character of an abbreviation capitalized, _unless_ it is a two letter abbreviation. hence why `BC` has both letters capitalized. This should be `BCProviderResetMfa`